pact/Pact/Core/Errors.hs

```diff
@@ -345,6 +345,7 @@ data EvalError
   | EnforcePactVersionParseFailure Text
   | RuntimeRecursionDetected QualifiedName
   | SPVVerificationFailure Text
+  | ContinuationError Text
```

Comment:
Should probably be renamed to `SPVContinuationError`?

Reply:
Yeah. The defpact cont errors are above, and this is presumably a proof function error from `yield/resume`.